src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -188,6 +188,7 @@ class MulticopterPositionControl : public control::SuperBlock
 		param_t xy_vel_d;
 		param_t xy_vel_max;
 		param_t xy_vel_cruise;
+		param_t xy_target_threshold;
```

Comment:
why does the parameter have a different name than it's handle?
you might also switch to BlockParam ;)